vendor/vendor.js

```diff
@@ -9,7 +9,7 @@
 
 var vendor = {
 	"moment.js" : "node_modules/moment/min/moment-with-locales.js",
-	"moment-timezone.js" : "node_modules/moment-timezone/moment-timezone.js",
+	"moment-timezone-with-data.js" : "node_modules/moment-timezone/builds/moment-timezone-with-data.js",
```

Comment:
This change is a bug introduced into commit a579b12448a99d1ba8f4bebd1ebb8a8397f3c406